plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/bedrock-runtime.ts

```diff
@@ -342,6 +358,62 @@ export class BedrockRuntimeServiceExtension implements ServiceExtension {
   ) {
     const { stopReason, usage } = response.data;
 
+    BedrockRuntimeServiceExtension.setStopReason(span, stopReason);
+    this.setUsage(response, span, usage, startTime);
+  }
+
+  private responseHookConverseStream(
+    response: NormalizedResponse,
+    span: Span,
+    tracer: Tracer,
+    config: AwsSdkInstrumentationConfig,
+    startTime: HrTime
+  ) {
+    return {
+      ...response.data,
+      stream: this.wrapConverseStreamResponse(
```

Comment:
Does `stream: this.wrapConverseStreamResponse(...)` overwrite the `stream` from `...response.data`?
I think it will be nice to have a comment to call this out.

Reply:
Yup - added a comment about it